src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -41,21 +44,126 @@ export const Breadcrumbs = ({
+   * 2) Finds the second last breadcrumb item for mobile Breadcrumbs back icon
+   * 3) Finds all the breadcrumb items between the first and last breadcrumb items so they can be placed in the dropdown.
+   */
+  const breadcrumbsItems = flattenBreadcrumbsItems(children); /* 1 */
+  /* 2 */
+  const backBreadCrumb =
+    breadcrumbsItems.length > 1
+      ? React.cloneElement(
+          breadcrumbsItems[
+            breadcrumbsItems.length - 2
+          ] as React.ReactElement<any>,
+          {
+            variant: 'back',
+          },
+        )
+      : null;
+  /* 3 */
+  const dropdownMenuItems = breadcrumbsItems
+    .slice(1, breadcrumbsItems.length - 1)
+    .map((breadcrumbItem, index) => {
+      const menuItem = breadcrumbItem as JSX.Element;
+      return (
+        <DropdownMenuItem
+          href={menuItem.props.href}
+          key={`breadcrumb-menu-item-${index}`}
+        >
+          {menuItem.props.text}
+        </DropdownMenuItem>
+      );
+    });
+
   const generatedId = useUID();
   const breadcrumbsId = id || generatedId;
 
-  const componentClassName = clsx(styles['breadcrumbs'], className, {});
-
+  const componentClassName = clsx(styles['breadcrumbs'], className);
   return (
     <nav
       aria-label={ariaLabel}
       className={componentClassName}
       id={breadcrumbsId}
       {...other}
     >
-      <ul className={styles['breadcrumbs__list']}>{children}</ul>
+      {/**
+       * 1) Back icon breadcrumb always exists, just hidden via css depending on breakpoint to increase performance
+       * 2) The ellipsis breadcrumb with dropdown only exists if there would be overflow and there are 3 or more breadcrumb items
+       * 3) If (2) conditions aren't met, display all breadcrumbs
+       */}
+      <ul className={styles['breadcrumbs__list']} ref={ref}>
+        {/* 1 */}
+        {backBreadCrumb}
+        {/* 2 */}
+        {shouldTruncate && breadcrumbsItems.length > 2 ? (
+          <>
+            {breadcrumbsItems[0]}
+            <BreadcrumbsItem
+              dropdownMenuItems={dropdownMenuItems}
+              href={null}
+              variant="collapsed"
+            />
+            {breadcrumbsItems[breadcrumbsItems.length - 1]}
+          </>
+        ) : (
+          /* 3 */
+          breadcrumbsItems
+        )}
+      </ul>
     </nav>
   );
 };
 
+const flattenBreadcrumbsItems = (children: React.ReactNode) => {
+  const flattenedChildren = flattenReactChildren(children);
+  /**
+   * Throws error if children are not BreadcrumbsItems
+   */
+  const shouldThrowError = (flattenedChildren as JSX.Element[]).some(
+    (child) => {
+      if (child.type === BreadcrumbsItem || child.type === Breadcrumbs.Item) {
+        return false;
+      }
+      return true;
+    },
+  );
+  if (process.env.NODE_ENV !== 'production' && shouldThrowError) {
+    throw 'Only <Breadcrumbs.Item>, <BreadcrumbsItem>, or React.Fragment of aforementioned components allowed';
```

Comment:
I hate to give you more work, but should we have a test for this?

Reply:
Nah, but good callout, will add test